package.json

```diff
@@ -36,7 +38,7 @@
     "husky": "^7.0.4",
     "lint-staged": "^12.3.2",
     "prettier": "2.5.1",
-    "typescript": "4.5.5",
+    "typescript": "^4.5.5",
```

Comment:
With the carat here--would likely recommend staying on same major and minor version for now.